bindings/netstandard/ElectionGuard/Tests/ElectionGuard.Tests/TestEncrypt.cs

```diff
@@ -0,0 +1,36 @@
object_id\":\"john-hancock-selection\",\"sequence_order\":2,\"candidate_id\":\"john-hancock\"},{\"object_id\":\"write-in-selection\",\"sequence_order\":3,\"candidate_id\":\"write-in\"}],\"ballot_title\":{\"text\":[{\"value\":\"Justice of the Supreme Court\",\"language\":\"en\"},{\"value\":\"Juez de la corte suprema\",\"language\":\"es\"}]},\"ballot_subtitle\":{\"text\":[{\"value\":\"Please choose up to two candidates\",\"language\":\"en\"},{\"value\":\"Uno\",\"language\":\"es\"}]},\"vote_variation\":\"n_of_m\",\"electoral_district_id\":\"jefferson-county\",\"name\":\"Justice of the Supreme Court\",\"primary_party_ids\":[\"whig\",\"federalist\"],\"number_elected\":2,\"votes_allowed\":2},{\"@type\":\"ReferendumContest\",\"object_id\":\"referendum-pineapple\",\"sequence_order\":1,\"ballot_selections\":[{\"object_id\":\"referendum-pineapple-affirmative-selection\",\"sequence_order\":0,\"candidate_id\":\"referendum-pineapple-affirmative\"},{\"object_id\":\"referendum-pineapple-negative-selection\",\"sequence_order\":1,\"candidate_id\":\"referendum-pineapple-negative\"}],\"ballot_title\":{\"text\":[{\"value\":\"Should pineapple be banned on pizza?\",\"language\":\"en\"},{\"value\":\"¿Debería prohibirse la piña en la pizza?\",\"language\":\"es\"}]},\"ballot_subtitle\":{\"text\":[{\"value\":\"The township considers this issue to be very important\",\"language\":\"en\"},{\"value\":\"El municipio considera que esta cuestión es muy importante\",\"language\":\"es\"}]},\"vote_variation\":\"one_of_m\",\"electoral_district_id\":\"harrison-township\",\"name\":\"The Pineapple Question\",\"number_elected\":1,\"votes_allowed\":1}],\"ballot_styles\":[{\"object_id\":\"jefferson-county-ballot-style\",\"geopolitical_unit_ids\":[\"jefferson-county\"]},{\"object_id\":\"harrison-township-ballot-style\",\"geopolitical_unit_ids\":[\"jefferson-county\",\"harrison-township\"]},{\"object_id\":\"harrison-township-precinct-east-ballot-style\",\"geopolitical_unit_ids\":[\"jefferson-county\",\"harrison-township\",\"harrison-township-precinct-east\",\"rutledge-elementary\"]},{\"object_id\":\"rutledge-elementary-ballot-style\",\"geopolitical_unit_ids\":[\"jefferson-county\",\"harrison-township\",\"rutledge-elementary\"]}],\"name\":{\"text\":[{\"value\":\"Jefferson County Spring Primary\",\"language\":\"en\"},{\"value\":\"Primaria de primavera del condado de Jefferson\",\"language\":\"es\"}]},\"contact_information\":{\"address_line\":[\"1234 Paul Revere Run\",\"Jefferson, Hamilton 999999\"],\"name\":\"Hamilton State Election Commission\",\"email\":[{\"annotation\":\"press\",\"value\":\"inquiries@hamilton.state.gov\"},{\"annotation\":\"federal\",\"value\":\"commissioner@hamilton.state.gov\"}],\"phone\":[{\"annotation\":\"domestic\",\"value\":\"123-456-7890\"},{\"annotation\":\"international\",\"value\":\"+1-123-456-7890\"}]},\"start_date\":\"2020-03-01T08:00:00-05:00\",\"end_date\":\"2020-03-01T20:00:00-05:00\",\"election_scope_id\":\"jefferson-county-primary\",\"type\":\"primary\"}";
```

Comment:
this is the same manifest in the data folder. i added it as a string so i didn't have to take a dependency to the file api's on each test platform